tools/build-overrides/build-overrides.sh

```diff
@@ -362,8 +358,7 @@ envs:
 - name: "CONTROLLER_DISTRIBUTION_ZIP"
   value: "${controller_distribution_zip}"
 artifacts:
-- name: "ADD_ONS_DISTRIBUTION_ZIP"
-  target: "add_ons_distribution.zip"
+- name: "rhpam_add_ons_distribution.zip"
```

Comment:
This controller is for RHDM (see https://github.com/jboss-container-images/jboss-kie-modules/pull/407/files#diff-34d9276ab4d4a89c1f08e5458bb8e87ab91a38c7e611a20f8381087b1e92c073R349)
```suggestion
- name: "rhdm_add_ons_distribution.zip"
```